cmd/operator-sdk/cli/legacy.go

```diff
@@ -73,7 +73,7 @@ func GetCLIRoot() *cobra.Command {
 		add.NewCmd(),
 		alpha.NewCmd(),
 		build.NewCmd(),
-		bundle.NewCmd(),
+		bundle.NewLegacyCmd(),
```

Comment:
May another option would be we keep the NewCmd and then do something as:

```
          if kbutil.HasProjectFile() {
		//todo actions
		return nil
	}

	// todo: remove when the legacy layout is no longer supported
	// note that the above if will no longer be required as well.
	if err := doLegacyX(..); err != nil {
		log.Fatalf("Failed to X %s: %v", n, err)
	}
	return nil
```

Reply:
You're suggesting using `kbutil.HasProjectFile()` in each `NewCmd()` that has both legacy and new behavior?